### PR TITLE
Use inline SVGs for Summaries

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
Refactor Summaries to use inline SVGs

### How to review 
Check lists work as expected and all examples look as expected

### CODE REVIEW

**Comment by rmccar** [comment]
[Use inline SVGs for Summaries](https://trello.com/c/MTW36mZU/620-use-inline-svgs-for-summaries)

File: src/components/summary/_macro.njk [hunk 1]
```diff
@@ -36,14 +36,21 @@
                                 {% for rowItem in row.rowItems %}
                                     <tr class="summary__row{{ " summary__row--has-values" if rowItem.valueList else "" }}">
                                         <td
-                                            class="summary__item-title{{ " summary__item-title--has-icon" if rowItem.icon else "" }}"
+                                            class="summary__item-title{{ " summary__item-title--has-icon" if rowItem.icon else "" }}{{ " summary__item-title--has-icon--check" if rowItem.icon == 'check' }}"
```
Comment (boxadesign) [inline src/components/summary/_macro.njk:39]: `summary__item-title--has-icon--check` doesn't look right as a class name.

Comment (boxadesign) [reply src/components/summary/_macro.njk:39]: Double use of `--` modifier.

File: src/components/summary/_macro.njk [hunk 2]
```diff
@@ -36,14 +36,21 @@
...
                                     <tr class="summary__row{{ " summary__row--has-values" if rowItem.valueList else "" }}">
                                         <td
-                                            class="summary__item-title{{ " summary__item-title--has-icon" if rowItem.icon else "" }}"
+                                            class="summary__item-title"
                                             {% if rowItem.rowTitleAttributes is defined %}{% for attribute, value in (rowItem.rowTitleAttributes.items() if rowItem.rowTitleAttributes is mapping and rowItem.rowTitleAttributes.items else rowItem.rowTitleAttributes) %}{{attribute}}="{{value}}" {% endfor %}{% endif %}
                                         >
                                             {% if rowItem.icon is defined %}
-                                                <span class="summary__icon icon--{{ rowItem.icon }}"></span>
+                                                {% from "components/icons/_macro.njk" import onsIcon %}
+                                                <span class= "summary__item-title--icon {% if rowItem.icon == 'check' %} summary__item-title--icon-check{% endif %}">
```
Comment (boxadesign) [inline src/components/summary/_macro.njk:44]: The class is a modifier of a parent item with the same class name.

**Pull request merged**